specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

```diff
@@ -1383,18 +1359,6 @@
     },
     "NetworkRuleSet": {
       "properties": {
-        "bypass": {
```

Comment:
Removing a property is a breaking change

Reply:
This parameter was accidentally added in the swagger. It doesn't exists in service.
So removing it is just a fix in swagger side.